src/icons/crypto/HI.tsx

```diff
@@ -0,0 +1,189 @@
+import SvgIcon, {type SvgIconProps} from '@mui/material/SvgIcon';
+import React from 'react';
+
+const Helium = React.forwardRef<
+  SVGSVGElement,
+  {
+    iconRef?: React.ForwardedRef<SVGSVGElement>;
+  } & SvgIconProps
+>(({iconRef, ...props}, ref) => {
+  return (
+    <SvgIcon viewBox="0 0 512 512" {...props} ref={iconRef || ref}>
+      <defs>
+        <clipPath id="clippath">
+          <circle fill="none" cx="256" cy="256" r="223.99" />
+        </clipPath>
+        <filter id="drop-shadow-1" filterUnits="userSpaceOnUse">
```

Comment:
Please consider the following:
1. Avoid raster graphics, it is better to take the SVG from their official website, and add some circle and change colors in Figma, to compose a simple icon which fits our standards (you can also ask Daniil)
2. Optimize the SVG code https://unstoppabledomains.slack.com/archives/C03702HJCN7/p1673957738038389

Reply:
That can work